modules/serverless-workflow-tag-revisions-unassign.adoc

```diff
@@ -0,0 +1,29 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="unassigning-tag-revisions_{context}"]
+= Unassigning tag revisions 
+
+Tags assigned to revisions in a traffic block can be unassigned. Unassigning tags removes the custom URLs. 
+
+[NOTE]
+====
+If a revision is untagged and it is assigned 0% of the traffic, it is removed from the traffic block entirely.
+====
+
+.Procedure
+* A user can unassign the tags for revisions using the `kn service update` command:
```

Comment:
I would be consistent in the wording for this step between unassigning and assigning tags. Prefer to start steps with an action verb, either:

Like either "Use the following command to assign a tag:" or "Assign a tag using the following command:"